### PR TITLE
Reloader

### PR DESCRIPTION
**Pull request by xezian**
This branch is wildly different from master

### CODE REVIEW

**Comment by xezian** [comment]
There's still an issue with yearOneImages I couldn't figure out, where it makes most but not all of the images. The error has to do with buffer stream output?  To work around it I copied the compiled image folder over from my working branch into the new location. Pull request contains hot-reloading, search by tag, topic or title, and basics of new logging setup for both front and back end.

**Comment by xezian** [comment]
screen shot of failing image build (only happens on about 10 of the images including this, the last one):
screen shot 2018-09-01 at 5 42 47 pm

**Comment by FotoVerite** [comment]
It's because it's going too fast and overloading imagemagick. Why we have a delay.

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,54 +1,121 @@
...
+                    )}
+      />
+
+      {/* or else display all songs */}
+      <Route
+        path="/"
+        render={props => (
+          <SearchBy
+            filterBy={false}
+            {...props}
```
Comment (FotoVerite) [inline client/src/components/App.jsx:113]: we should style wise always put path before props just for easier reading.

**Pull request closed without merging**